spec/mailers/previews/claims_api_submission_report_mailer_preview.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
```

Comment:
There is a way to run this preview, and I will let you know when I remember, unless you know.

Reply:
http://localhost:3000/rails/mailers/claims_api_monthly_report_mailer/build